hydro_lang/src/deploy/mod.rs

```diff
@@ -12,24 +12,22 @@ use stageleft::QuotedWithContext;
 pub mod macro_runtime;
 pub use macro_runtime::*;
 
-#[cfg(feature = "deploy")]
-#[cfg(stageleft_runtime)]
+#[cfg(all(stageleft_runtime, feature = "deploy"))]
```

Comment:
Unfortunately the parser for `stageleft` looks for just `cfg(stageleft_runtime)` on its own and doesn't reason about `all(...)`. For now we can probably just revert these changes but can create an issue for finding a more robust solution.

Reply:
Will revert

Reply:
https://github.com/hydro-project/stageleft/issues/6